src/PlatformInstaller/Views/ShellViewModel.cs

```diff
@@ -119,6 +125,12 @@ void RunStartupSequence()
                 }
             }
         }
+
+        if (!runtimeUpgradeManager.Is452orLaterInstalled())
```

Comment:
perhaps invert this if? makes for easier reading if u dont need to notice the negation